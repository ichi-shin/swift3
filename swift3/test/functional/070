#!/bin/bash
# Copyright(c)2014 NTT corp. All Rights Reserved.
# DB1-1-3-3_3

. ./common

echo '<LifecycleConfiguration>
  <Rule>
    <ID>Rule-1</ID>
    <Prefix>a</Prefix>
    <Status>Enabled</Status>
    <Expiration>
      <Days>10</Days>
    </Expiration>
  </Rule>
</LifecycleConfiguration>' > $tmp.lifecycle.xml

S3USER=tester

_s3_put /bucket
MD5=$(_md5 $tmp.lifecycle.xml) _s3_put /bucket?lifecycle -T $tmp.lifecycle.xml
_s3_delete /bucket?lifecycle
_s3_get /bucket?lifecycle -D - | _filter_curl xml
