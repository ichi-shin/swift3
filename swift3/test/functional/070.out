QA output created by 070
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester --contentMd5 B3TydZQCrG9WflISDhI5SQ== -- -X PUT -T /TMP.lifecycle.xml http://SWIFT_HOST/bucket?lifecycle... 200
> s3curl --id tester -- -X DELETE http://SWIFT_HOST/bucket?lifecycle... 204
> s3curl --id tester -- -X GET -D - http://SWIFT_HOST/bucket?lifecycle... 404
HTTP/1.1 404 Not Found
Content-Type: text/xml
Date: DATE
Transfer-Encoding: chunked
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<Error>
  <Code>NoSuchLifecycleConfiguration</Code>
  <Message>The lifecycle configuration does not exist. .</Message>
  <RequestId>TXID</RequestId>
</Error>
